src/styles.js

```diff
@@ -0,0 +1,137 @@
+import styled from 'styled-components';
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  html, body {
+    height: 100%;
+  }
+
+  body {
+    font-family: Arial, sans-serif;
+    color: black;
+  }
+`;
+
+export const MainText = styled.div`
+font-size: 50px;
+font-weight: 600;
+`
+
+export const Line = styled.div`
+background-color: black;
+width: 1.5px;
+height: 100px;
+margin-bottom: 10px;
+`
+
+export const AppContainer = styled.div`
+  margin-left: 70px;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  flex-direction: column;
+`;
+
+export const Main = styled.main`
+  height: 400px;
+  width: 400px;
+`;
+
+
+export const TodoContainer = styled.section`
+  height: 400px;
+  border-color: black;
+  border-radius: 10px;
+  border-width: 2px;
+  border-style: solid;
+  padding: 20px;
+`;
+
+export const TodoList = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin-bottom: 10px;
+`;
+
+export const TodoItem = styled.li`
+  background-color: aliceblue;
+  padding: 10px;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  display: flex;
+  justify-content: space-between;
+  word-break: break-all;
+`;
+
+export const TodoInput = styled.div`
+  display: flex;
+  justify-content: space-between;
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 1px;
+  margin-bottom: 10px;
+`;
```

Comment:
<img width="440" alt="스크린샷 2024-09-23 오후 5 21 11" src="https://github.com/user-attachments/assets/0f0a455b-64c1-43bc-87fd-d5b90fc631a5">

`focus` 상태일 때 인풋박스 `outline`은 없애주기로 해요 🌞
```suggestion
export const TodoInput = styled.div`
  display: flex;
  justify-content: space-between;
  border-radius: 10px;
  border-style: solid;
  border-width: 1px;
  margin-bottom: 10px;
  &:focus {
    outline: none; 
  }
`;
```